src/components/add-button/add-button.js

```diff
@@ -0,0 +1,11 @@
+import './add-button.css'
+
+function AddButton () {
+    return (
+        <>
+            <button className={'add_button'}>+ ADD MOVIE</button>
+        </>
```

Comment:
1. Do we really need an empty fragment?
2. Do we really need a single component for an add button? Maybe we can pass the title and the specific class name to the single button component. What do you think?

```suggestion
            <button className={'add_button ' + props.buttonClassName}>{props.title}</button>
```

Reply:
Done